index.html

```diff
@@ -1,14 +1,511 @@
 <!DOCTYPE HTML>
 <html>
   <head>
-    <title>Mockup Home</title>
+    <title>Big Jim's Landing Pages</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    <link rel="stylesheet" href="css/remock.css">
+    <!--Text Font-->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <!--Header Font-->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@900&display=swap" rel="stylesheet"> 
+    <!--Icon Font-->
+    <script src="https://kit.fontawesome.com/5e40fbdb20.js" crossorigin="anonymous"></script>
   </head>
+
   <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
-</html>
\ No newline at end of file
+   
+      <!---Header-->
+    <section id="header">
+      <a href="/index.html"><h1>BIG JIM'S BEE BLOG</h1></a>
+      <h5>FOLLOW ALONG FOR SOME FUCKING FUN</h5>
+      <hr>
+      <hr>
+    </section>
+
+
+    <!--Feature-->
+    <section id="features">
+      <h3>TO THE MOON <span class="bold">BACK...</span></h3>
+      <!--Display Box => Flex/ 3 columnns-->
+      <div class="flex-box">
+        <div class="features-flex-left">
+          <img src="Pictures/bee-movie01.jpg" alt="Bee Movie">
+          <h5>FORKLIFT CERTIFIED</h5>
+        </div>
+        <div class="features-flex-center">
+          <img src="Pictures/bee-movie02.jpg" alt="Bee Movie">
+          <h5>GIMME A CHOPPED CHEEZ</h5>
+        </div>
+
+        <div class="features-flex-right">
+          <img src="Pictures/bee-movie03.jpg" alt="Bruh">
+          <h5>T-POSE SON</h5>
+        </div>
+      </div>
+      
+      <!--Learn More Button-->
+      <button class="button-pink">
+            <i class="far fa-file"></i>
+            <a href="/learn-more.html">More. MOOOOOOORE.</a>
+        </button>
+    </section>

```

Comment:
Header styling is great! This section of the page is right on and a great match compared to the remock project screenshots. This header section also responds well to different window sizes.